### ISSUE TITLE
Server endpoint is listed incorrect

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/memgator#4]
I am running the OS X binary via `./memgator-darwin-amd64 server`. The TimeMap endpoint echoed is `http://localhost:1208/timemap/timemap/link|json|cdxj/{URI-R}`. http://localhost:1208/timemap/timemap/json/matkelly.com from a client returns the following on the client:
```
TimeMap  : http://localhost:1208/timemap/timemap/link|json|cdxj/{URI-R}
TimeGate : http://localhost:1208/timegate/timegate/{URI-R} [Accept-Datetime]
TimeNav  : http://localhost:1208/timenav/link|json|cdxj/{YYYY[MM[DD[hh[mm[ss]]]]]}/{URI-R}
Redirect : http://localhost:1208/redirect/{YYYY[MM[DD[hh[mm[ss]]]]]}/{URI-R}
```
Removing the redundant "timemap" in the URI returns the expected results, e.g., `curl http://localhost:1208/timemap/json/matkelly.com` returns the following to the client:
```
{
  "original_uri": "matkelly.com",
  "mementos": {
    "list": [
      {
        "datetime": "2006-05-14T12:35:11Z",
        "uri": "http://web.archive.org/web/20060514123511/http://www.matkelly.com/"
      },
      {
        "datetime": "2006-05-16T21:38:52Z",
...
```

Should the server be listing `TimeMap  : http://localhost:1208/timemap/link|json|cdxj/{URI-R}` and not `TimeMap  : http://localhost:1208/timemap/timemap/link|json|cdxj/{URI-R}`? Does the same thing apply to the `timegate/timegate` redundancy?

**Comment by ibnesayeed** [comment]
Thanks for reporting, I actually identified that issues #3 and fixed it 2648471 before you reported. The new binary files were silently updated in the same release (with a small note). It was a minor issue, hence I did not initiate a new release for it. If you download the binaries again, this should not happen.

**Comment by machawk1** [comment]
Verified as fixed.
